examples/fodo.py

```diff
@@ -7,35 +7,35 @@
 sys.path.append(os.path.abspath(os.path.join(os.path.dirname(__file__), "../src/")))
 
 from pals_schema.MagneticMultipoleParameters import MagneticMultipoleParameters
-from pals_schema.DriftElement import DriftElement
-from pals_schema.QuadrupoleElement import QuadrupoleElement
+from pals_schema.Drift import Drift
+from pals_schema.Quadrupole import Quadrupole
```

Comment:
Perfect, thank you!

In a follow-up PR, can you please check if we want to write simply
```py
from pals import Drift, Quadrupole, Beamline
```

If empty/specific `__init__.py` files are missing we can add them.